float32 ee_z_vel